tesseract_core/sdk/tesseract.py

```diff
@@ -85,6 +85,9 @@ def from_image(
         user: str | None = None,
         input_path: str | Path | None = None,
         output_path: str | Path | None = None,
+        output_format: Literal[
+            "json", "json+base64", "json+binref", "msgpack"
```

Comment:
ohh, if msgpack is deprecated, we should probably remove this?
```suggestion
            "json", "json+base64", "json+binref"
```

Reply:
Good catch, thanks!